chrome/content/zotero/xpcom/quickCopy.js

```diff
@@ -321,32 +331,35 @@ Zotero.QuickCopy = new function() {
 				}
 				
 				var content = {
-					text: contentType == "html" ? html : text,
+					text: format.contentType == "html" ? html : text,
 					html: copyHTML
 				};
 				
 				return content;
 			}
 			
+			// determine locale preference
+			var locale = format.locale ? format.locale : Zotero.Prefs.get('export.quickCopy.locale');
+			
 			// Copy citations if shift key pressed
 			if (modified) {
-				var csl = Zotero.Styles.get(format).getCiteProc();
+				var csl = Zotero.Styles.get(format.id).getCiteProc(locale);
 				csl.updateItems([item.id for each(item in items)]);
 				var citation = {citationItems:[{id:item.id} for each(item in items)], properties:{}};
 				var html = csl.previewCitationCluster(citation, [], [], "html"); 
 				var text = csl.previewCitationCluster(citation, [], [], "text"); 
 			} else {
-				var style = Zotero.Styles.get(format);
-				var cslEngine = style.getCiteProc();
+				var style = Zotero.Styles.get(format.id);
+				var cslEngine = style.getCiteProc(locale);
  				var html = Zotero.Cite.makeFormattedBibliographyOrCitationList(cslEngine, items, "html");
 				var text = Zotero.Cite.makeFormattedBibliographyOrCitationList(cslEngine, items, "text");
 			}
 			
-			return {text:(contentType == "html" ? html : text), html:html};
+			return {text:(format.contentType == "html" ? html : text), html:html};
 		}
 		
-		throw ("Invalid mode '" + mode + "' in Zotero.QuickCopy.getContentFromItems()");
-	}
+		throw ("Invalid mode in Zotero.QuickCopy.getContentFromItems()");
```

Comment:
Taken care of in https://github.com/rmzelle/zotero/commit/3a83b870a750e429b727a01ccf1c5cd7ceb4b94b